File: app/src/main/java/com/monk/retrofit/netapi/HttpApi.java
package com.monk.retrofit.netapi;

import io.reactivex.Observable;
import okhttp3.ResponseBody;
import retrofit2.http.GET;
import retrofit2.http.Header;
import retrofit2.http.Path;

/**
 * @author monk
 * @date 2019-02-19
 */
public interface HttpApi {
    /**
     *  http://223.113.1.208:63080/AppWebApi/api/employee/xx/coiper/list
     * @param accessCode
     * @param sign
     * @param unitGuid
     * @return
     */
    @GET("employee/{unitGuid}/coiper/list")
    Observable<ResponseBody>getDataForMap(@Header("AccessCode") String accessCode, @Header("Sign") String sign, @Path("unitGuid") String unitGuid);
}

File: module_fragment/src/main/java/com/monk/modulefragment/retrofit/netapi/HttpApi.kt
package com.monk.modulefragment.retrofit.netapi

import io.reactivex.Observable
import okhttp3.ResponseBody
import retrofit2.http.GET
import retrofit2.http.Header
import retrofit2.http.Path

/**
 * @author monk
 * @date 2019-02-19
 */
interface HttpApi {
    /**
     * http://223.113.1.208:63080/AppWebApi/api/employee/xx/coiper/list
     * @param accessCode
     * @param sign
     * @param unitGuid
     * @return
     */
    @GET("employee/{unitGuid}/coiper/list")
    fun getDataForMap(@Header("AccessCode") accessCode: String?, @Header("Sign") sign: String?, @Path("unitGuid") unitGuid: String?): Observable<ResponseBody?>?
}